rangers-api/Gindows/Control.h

```diff
@@ -0,0 +1,175 @@
+        csl::ut::MoveArray<void*> qword1E0;
+        csl::ut::MoveArray<void*> qword200;
+        csl::ut::MoveArray<void*> qword220;
+        csl::ut::MoveArray<void*> qword240;
+        csl::ut::MoveArray<void*> qword260;
+        csl::ut::MoveArray<void*> qword280;
+        csl::ut::MoveArray<void*> qword2A0;
+        csl::ut::MoveArray<void*> qword2C0;
+        csl::ut::MoveArray<void*> qword2E0;
+        csl::ut::MoveArray<void*> qword300;
+        csl::ut::MoveArray<void*> qword320;
+        csl::ut::MoveArray<void*> qword340;
+        csl::ut::MoveArray<void*> qword360;
+        csl::ut::MoveArray<void*> qword380;
+        csl::ut::MoveArray<void*> qword3A0;
+        csl::ut::MoveArray<void*> qword3C0;
+        csl::ut::MoveArray<void*> qword3E0;
+        csl::ut::MoveArray<void*> qword400;
+        csl::ut::MoveArray<void*> qword420;
+        csl::ut::MoveArray<void*> qword440;
+        csl::ut::MoveArray<void*> qword460;
+        csl::ut::MoveArray<void*> qword480;
+        csl::ut::MoveArray<void*> qword4A0;
+        csl::ut::MoveArray<void*> qword4C0;
+        int64_t flags4E0; // flags, see ctor
+        Control* qword4E8; // see sub_14114c860, sub_14114e0e0
+        UnkStr0 unkStr0;
+        UnkStr1 unkStr1;
+        UnkStr2 unkStr2;
+        int64_t qword658;
+        int64_t qword660;
+        int64_t oword668;
+        int64_t oword670;
+        int64_t qword678;
+        int64_t qword680;
+        int64_t qword688;
+        int dword690;
+        int64_t qword694;
+        int dword69C;
+        int64_t qword6A0;
+        int64_t qword6A8;
+        int dword6B0;
+        UnkStr1 qword6B8;
+        int dword708;
+        char byte70C;
+        void* qword710;
+        UnkStr3 unkStr3;
+
+        virtual void* GetInfo() override;
+        virtual Object* UnkFunc(char unk) override;
+        virtual void CompUnkFunc() override;
+
+        virtual void ContUnkFuncNull() {};
+        virtual void ContUnkFunc0(int* unk);
+        virtual void ContUnkFunc1();
+        virtual void* ContUnkFunc2();
+        virtual void ContUnkFunc3();
+        virtual char ContUnkFunc4(int* unk);
+        virtual void* ContUnkSet0(void* unk);
+        virtual void* ContGetQWORD680(void* unk);
+        virtual void* ContSetQWORD680(void* unk);
+        virtual void* ContGetQWORD688(void* unk);
+        virtual void* ContSetQWORD688(void* unk);
+        virtual void* ContUnkFunc5(void* unk);
+        virtual void* ContSet1(void* unk);
+        virtual void* ContSet2(void* unk);
+        virtual char ContUnkFunc6(void* unk);
+        virtual char ContUnkFunc7();
+        virtual void* ContUnkFunc8(); // something with qword420
+        virtual void* ContUnkFunc9(float unk);
+        virtual void* ContUnkFunc10(); // something with qword40
+        virtual void* ContUnkFunc11(); // something with qword60
+        virtual void* ContUnkFunc13(); // something with qword220
+        virtual void* ContUnkFunc14(); // something with qword240
```

Comment:
The reason why I use uint64_t as a return value for unknown functions is because void* has the tendency to generate disassemblies where an access of offset 0x18 for example looks like `foo[24]` or something similar, anyway it's harder to read than when the disassembler has no info at all, then it generates things like `*(foo + 24)`.

Don't need to change this everywhere, just for future reference.